### PR TITLE
Fix PDF Date

### PR DESCRIPTION
**Pull request by rogeruiz**
This PR adds `data/*toml` files to the repository to better manage the PDFs and their updated dates on the homepage. These files can be used to manage more content around the PDFs, and can be useful for other things in the future as well.

Here's a screenshot with the buttons generated form the `data/*toml` files. 
screen shot 2016-03-07 at 2 12 20 pm

I will be including line-notes pointing to the relevant bits in this PR.

[More information on `data` in Hugo can be found here](http://gohugo.io/extras/datafiles/)

### CODE REVIEW

File: data/ResearchReport.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+File = "files/FFD_ResearchReport_0316.pdf"
+Date = "3/16"
```
Comment (rogeruiz) [inline data/ResearchReport.toml:2]: A `Date` property exists in the `toml` file.

File: layouts/index.html [hunk 2]
```diff
@@ -90,12 +90,12 @@ <h3 class="usa-graphic-list-heading">Civic engagement</h3>
         </div>
         <div class="usa-width-two-thirds">
           <h3 class="usa-font-lead">All our work is guided by user research — structured conversations with people from varied populations. This research report highlights our lines of inquiry, provides detailed descriptions of what we learned, and raises questions that warrant further study.</h3>
-          <a class="usa-button usa-button-secondary" href="{{ .Site.BaseURL }}files/FFD_ResearchReport_0316.pdf" onclick="ga('send', 'event', 'Downloaded PDF report', 'Clicked download of report');">Download the report (PDF)</a>
-          <span class="usa-text-small">Updated 2/1/16</span>
+          <a class="usa-button usa-button-secondary" href="{{ .Site.BaseURL }}{{ .Site.Data.ResearchReport.File }}" onclick="ga('send', 'event', 'Downloaded PDF report', 'Clicked download of report');">Download the report (PDF)</a>
+          <span class="usa-text-small">Updated {{ .Site.Data.ResearchReport.Date }}</span>
```
Comment (rogeruiz) [inline layouts/index.html:94]: The `Date` property is retrieved in the `go` template.

**Comment by rogeruiz** [comment]
@ericadeahl Can you take a look at the attached screenshot :point_up: and let me know if those dates are accurate? The _Methodology supplement PDF_ originally had a date like `2/1/16`. I updated it to look more like the date [you mentioned here](https://github.com/18F/ffd-microsite/pull/50#commitcomment-16541623).

**Comment by ericadeahl** [comment]
@rogeruiz :+1: looks good, thanks!

**Comment by juliasolorzano** [comment]
:+1:

**Pull request merged**